app-config.yaml

```diff
@@ -321,20 +290,21 @@ enabled:
   jenkins: ${JENKINS_ENABLED}
   permission: ${PERMISSION_ENABLED}
   azureAd: ${AZURE_AD_ENABLED}
+  bitbucketDiscovery: ${BITBUCKET_DISCOVERY_ENABLED}
 
 # Plugin Confluence
 confluence:
   # Confluence base URL for wiki API
   # Typically: https://{org-name}.atlassian.net/wiki
-  wikiUrl: ${CONFLUENCE_URL}
+  wikiUrl: https://wiki.atlassian.net
```

Comment:
They plan to use confluence.  Why did we remove the envs and hardcode dummy values?

Reply:
done